server/src/controllers/trip.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Request, Response } from 'express';
+
+const Trip = require('../models/trip.model.ts');
+
+exports.getAllTrips = async (req: Request, res: Response) => {
+  try {
+    const response = await Trip.find();
+    res.json(response);
+    res.status(200);
+  } catch (e) {
+    console.log(e);
+    res.status(500);
+  }
+};
+
+exports.addTrip = async (req: Request, res: Response) => {
+  try {
+    const { stops, label, description } = req.body;
+    //Is this line needed?
```

Comment:
Technically not needed at this point, though when we add server-side validation of the data, then it may be. No harm to leave it in